core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala

```diff
@@ -129,11 +129,11 @@ object Main extends {
     Option(
       client
         .apiextensions()
-        .v1beta1()
+        .v1()
         .customResourceDefinitions()
         .withName(App.ResourceName)
         .get()) match {
-      case Some(crd) if crd.getSpec.getVersion == App.GroupVersion =>
+      case Some(crd) if crd.getSpec.getVersions().contains(App.GroupVersion) =>
```

Comment:
This match is "weaker" now... not as accurate.
Should we match for "contains only"?

Reply:
was incorrect -> now checking if `getVersions` contains a `CustomResourceDefinitionVersion` with the appropriate name.